tools/ci/od_lints.dm

```diff
@@ -0,0 +1,32 @@
+//1000-1999
+#pragma FileAlreadyIncluded error
+#pragma MissingIncludedFile error
+#pragma MisplacedDirective error
+#pragma UndefineMissingDirective error
+#pragma DefinedMissingParen error
+#pragma ErrorDirective error
+#pragma WarningDirective error
```

Comment:
What's the actual use-case for this?

Reply:
https://github.com/tgstation/tgstation/pull/81892#discussion_r1518465169

Reply:
People compiling OpenDream outside of CI will receive errors since `#warn You are building with OpenDream. Remember to build TGUI manually.` is emitted and this elevates it to an error.

Reply:
Probably not worth erroring then yeah